### PR TITLE
Login close action added.

### PR DESCRIPTION
**Pull request by DineshKachhot**
Login close button action added.

### CODE REVIEW

File: Papr/Scenes/Login/LoginViewModel.swift [hunk 1]
```diff
@@ -81,6 +82,14 @@ class LoginViewModel: LoginViewModelInput, LoginViewModelOuput, LoginViewModelTy
         }
     }()
     
+    // MARK: Action
+    lazy var closeAction: CocoaAction = {
+        return CocoaAction { [unowned self] _ in
+//            self.authenticate()
```
Comment (jdisho) [inline Papr/Scenes/Login/LoginViewModel.swift]: We can remove unnecessary comments.

Comment (jdisho) [reply Papr/Scenes/Login/LoginViewModel.swift]: Why not returning just this: `self.sceneCoordinator.pop(animated: true)`? 
`sceneCoordinator.pop(animated: ...)` is an `Observable<Void>`

File: Papr/Scenes/Login/LoginViewModel.swift [hunk 2]
```diff
@@ -109,6 +118,12 @@ class LoginViewModel: LoginViewModelInput, LoginViewModelOuput, LoginViewModelTy
         }
         return .empty()
     }
+    
+    private func close() -> Observable<Void> {
```
Comment (jdisho) [inline Papr/Scenes/Login/LoginViewModel.swift]: ... and we can remove this

File: Papr/Scenes/Login/LoginViewController.swift [hunk 3]
```diff
@@ -35,6 +36,8 @@ class LoginViewController: UIViewController, BindableType {
         let outputs = viewModel.outputs
 
         loginButton.rx.action = inputs.loginAction
+
```
Comment (jdisho) [inline Papr/Scenes/Login/LoginViewController.swift]: remove the unnecessary line

**Review by jdisho — CHANGES_REQUESTED** [review]

File: Papr/Info.plist [hunk 4]
```diff
@@ -28,6 +28,14 @@
 				<string>papr</string>
 			</array>
 		</dict>
+		<dict>
+			<key>CFBundleTypeRole</key>
+			<string>Editor</string>
+			<key>CFBundleURLSchemes</key>
+			<array>
+				<string>papr://unsplash</string>
```
Comment (DineshKachhot) [inline Papr/Info.plist:36]: The general Url scheme.

**Pull request merged**